api-schema/src/main/java/org/stellar/anchor/api/custody/fireblocks/TransactionStatus.java

```diff
@@ -15,5 +17,13 @@ public enum TransactionStatus {
   CANCELLED,
   REJECTED,
   BLOCKED,
-  FAILED
+  FAILED;
+
+  public boolean isSuccessStatus() {
+    return this.equals(COMPLETED);
+  }
+
+  public boolean isObservableStatus() {
```

Comment:
I think, that we can remove `Status` ending from both methods, since Enum has it in it's name